docs/api/README.md

```diff
@@ -0,0 +1,212 @@
+
+### Wiki article "influenza" on 2015w01
+
+https://delphi.midas.cmu.edu/epidata/api.php?source=wiki&articles=influenza&epiweeks=201501
+
+    {"result":1,"epidata":[{"article":"influenza","count":30819,"hour":-1,"epiweek":201501}],"message":"success"}
+
+
+# Code Samples
+
+Libraries are available for [CoffeeScript](src/client/delphi_epidata.coffee), [JavaScript](src/client/delphi_epidata.js), [Python](src/client/delphi_epidata.py), and [R](src/client/delphi_epidata.R). The following samples show how to import the library and fetch national FluView data for epiweeks `201440` and `201501-201510` (11 weeks total).
+
+### CoffeeScript (in Node.js)
+
+````coffeescript
+# Import
+{Epidata} = require('./delphi_epidata')
+# Fetch data
+callback = (result, message, epidata) ->
+  console.log(result, message, epidata?.length)
+Epidata.fluview(callback, ['nat'], [201440, Epidata.range(201501, 201510)])
+````
+
+### JavaScript (in a web browser)
+
+````html
+<!-- Imports -->
+<script src="jquery.js"></script>
+<script src="delphi_epidata.js"></script>
+<!-- Fetch data -->
+<script>
+  var callback = function(result, message, epidata) {
+    console.log(result, message, epidata != null ? epidata.length : void 0);
+  };
+  Epidata.fluview(callback, ['nat'], [201440, Epidata.range(201501, 201510)]);
+</script>
+````
+
+### Python
+
+Optionally install the package using pip(env):
+````bash
+pip install delphi-epidata
+````
+
+Otherwise, place `delphi_epidata.py` from this repo next to your python script.
+
+````python
+# Import
+from delphi_epidata import Epidata
+# Fetch data
+res = Epidata.fluview(['nat'], [201440, Epidata.range(201501, 201510)])
+print(res['result'], res['message'], len(res['epidata']))
+````
+
+### R
+
+````R
+# Import
+source('delphi_epidata.R')
+# Fetch data
+res <- Epidata$fluview(list('nat'), list(201440, Epidata$range(201501, 201510)))
+cat(paste(res$result, res$message, length(res$epidata), "\n"))
+````
+
+# Related work
+
+ - Cook, Samantha, et al. "Assessing Google flu trends performance in the United States during the 2009 influenza virus A (H1N1) pandemic." PloS one 6.8 (2011): e23610.
+ - Broniatowski, David A., Michael J. Paul, and Mark Dredze. "National and local influenza surveillance through Twitter: an analysis of the 2012-2013 influenza epidemic." (2013): e83672.
+ - Dredze, Mark, et al. "HealthTweets. org: A Platform for Public Health Surveillance using Twitter." AAAI Conference on Artificial Intelligence. 2014.
+ - Generous, Nicholas, et al. "Global disease monitoring and forecasting with Wikipedia." (2014): e1003892.
+ - Hickmann, Kyle S., et al. "Forecasting the 2013–2014 Influenza Season Using Wikipedia." (2015): e1004239.
+ - McIver, David J., and John S. Brownstein. "Wikipedia usage estimates prevalence of influenza-like illness in the United States in near real-time." PLoS Comput Biol 10.4 (2014): e1003581.
```

Comment:
im realizing that this really needs to be updated, but i think i'll have to let delphi handle that